go/detectparser/parser.go

```diff
@@ -44,6 +45,22 @@ type RateLimitable struct {
 	RateLimitCache *cache.Cache
 }
 
+type parser struct {
+	RateLimitable
+	TenantID int
+}
+
+func setupParser(config *PfdetectConfig) parser {
+	if config == nil {
+		return parser{RateLimitable: RateLimitable{}, TenantID: 1}
+	}
+
+	return parser{
+		RateLimitable: NewRateLimitable(config.RateLimit),
+		TenantID:      config.TenantID,
```

Comment:
We should ship an upgrade script to ensure tenant_id will always be set in the configuration

Reply:
ok